src/runtime/step_runner.js

```diff
@@ -8,7 +8,7 @@ const {beginTiming, endTiming} = Time
 
 async function run({attachmentManager, defaultTimeout, scenarioResult, step, stepDefinition, transformLookup, world}) {
   beginTiming()
-  const parameters = stepDefinition.getInvocationParameters({scenarioResult, step, transformLookup})
+  const parameters = await Promise.all(stepDefinition.getInvocationParameters({scenarioResult, step, transformLookup}))
```

Comment:
Ah yes - hadn't thought of that! Want me to add a test for that or would you prefer to do that yourself?